Geometry/EcalAlgo/test/EcalPreshowerParameterDump.cc

```diff
@@ -42,9 +42,9 @@ EcalPreshowerCellParameterDump::EcalPreshowerCellParameterDump(const edm::Parame
 
   if (debug_) {
     edm::Service<TFileService> fs;
-    for (int iz = 0; iz < 2; ++iz) {
-      int zside = 2 * iz - 1;
-      for (int lay = 1; lay <= 2; ++lay) {
+    for (short iz = 0; iz < 2; ++iz) {
+      short zside = 2 * iz - 1;
+      for (short lay = 1; lay <= 2; ++lay) {
```

Comment:
@smuzaffar - `sprintf` is not thread safe - it should be replaced with` std::ostringstream`:

```c++
#include <sstream> // for ostringstream

std::ostringstream out;

out << ”hist” << iz << lay;
```

Reply:
Or even better to `std::format` when we switch to C++20:
```c++
std::string message = std::format("The answer is {}.", 42);
```
https://en.cppreference.com/w/cpp/utility/format

Reply:
done , used `ostringstream`